test/specs/equal.spec.js

```diff
@@ -147,7 +147,12 @@ describe('equal', function () {
     });
   });
 
+  describe('exclude = ["equal"]', function () {
```

Comment:
I think test of this option should be in `options.spec` as it relates to all checks, not only equal. 
The test itself should check 'x === y' under different option values: 
* exclude: ['equal']
* exclude: ['add']
* exclude: ['add', 'equal']
* exclude: []
* exclude: ['foo']